src/02-data-types.js

```diff
@@ -22,7 +24,9 @@ function getLengthOfString(str) {}
  *  convertToNumber("111");
  *  //> 111
  */
-function convertToNumber(val) {}
+function convertToNumber(val) {
+  return Number (val)
```

Comment:
When using constructors like `Number` we actually want the ` () ` to not be spaced out. We read this as one call to the `Number` constructor with the arguments passed.